doc/changes/latest.inc

```diff
@@ -45,6 +45,8 @@ Enhancements
 
 - Add projections when printing a :class:`mne.Info` in the notebook (:gh:`9403` by `Alex Gramfort`_)
 
+- Add "show_real_time" to :meth:`mne.io.Raw.plot` and :meth:`mne.preprocessing.ICA.plot_sources` to display acquisition time on x-axis (:gh:`9419` by `Martin Schulz`_)
```

Comment:
I have the feeling that there should be a better parameter name than `show_real_time`, but currently I can't think of a better name. Maybe someone else has a suggestion?

Reply:
Maybe `time_ticks` or `clock_ticks`?

Reply:
some thoughts:

time_axis = 'relative' | 'absolute'

or

time_axis = 'datetime' | ???